templates/email/order-cancel-failed.hypr

```diff
@@ -0,0 +1,22 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+<!--- Header --->
+<div class="mz-b2b-email-header">  
```

Comment:
Why is this using the B2B email header? This is not a B2B email. Use the general header for normal emails.

Reply:
I checked non b2b templates there we are not using header so I removed header html part.